### ISSUE TITLE
Task build fails due to ffi changes

### ISSUE THREAD

**Issue by JeromeK13** [hiero-ledger/hiero-sdk-rust#493]
### Description

Task build fails

```sh
task build
task: [rust:build] cargo build --message-format=short --features=ffi
task: [rust:swift-build] rustup component add rust-src 2> /dev/null
task: [rust:swift-build] rustup target add x86_64-apple-darwin 2> /dev/null
error: none of the selected packages contains these features: ffi
task: Failed to run task "rust:build": exit status 101
task: Failed to run task "rust:build": exit status 101
```

### Steps to reproduce

```sh
task build
```

### Hedera network

other

### Version

V.0.13.0

### Operating system

macOS

**Comment by izik1** [comment]
Hi! For now you can just `cargo build`, after this release that'll be fixed (specifically #491)

**Comment by JeromeK13** [comment]
Hey @izik1 
Thanks that worked after I manually installed protobuf and cloned the sub module(protobufs) as well.

KR
Jerome
